internal_transfers/actions/src/accounting.ts

```diff
@@ -104,11 +104,13 @@ interface SimulationPair {
 async function simulateBoth(
   simulator: TransactionSimulator,
   params: SettlementSimulationParams,
-  numAttempts: number = 2
+  numAttempts: number = 3
 ): Promise<SimulationPair> {
-  let attempts = 0;
-  while (attempts < numAttempts) {
+  const startBlock = params.common.blockNumber;
+  let attemptNumber = 0;
+  while (attemptNumber < numAttempts) {
     try {
+      params.common.blockNumber += attemptNumber;
```

Comment:
Mutating input params? 😰

Reply:
I can rework this.... 😄

Reply:
Fixed [003b4a1](https://github.com/cowprotocol/solver-rewards/pull/261/commits/003b4a18ef0dca10c2b20a717998a0ac86c7216c)